docs/wfc3tools/wf3cte.rst

```diff
@@ -1,22 +1,62 @@
+********
+`wf3cte`
+********
+
+The charge transfer efficiency (CTE) of the UVIS detector has inevitably 
+been declining over time as on-orbit radiation damage creates charge traps 
+in the CCDs. Faint sources in particular can suffer large flux losses or 
+even be lost entirely if observations are not planned and analyzed carefully. 
+The CTE loss will depend on the morphology of the source, the distribution of 
+electrons in the field of view (from sources, background, cosmic rays, and 
+hot pixels) and the population of charge traps in the detector column between 
+the source and the transfer register. The magnitude of the CTE loss increases 
+continuously with time as new charge traps form.
+
+CTE is typically measured as a pixel-transfer efficiency and would be unity for a 
+perfect CCD. One indicator of CTE is the Extended Pixel Edge Response (EPER). 
+Inefficient transfer of electrons in a flat-field exposure produces an exponential 
+tail of charge in the overscan region. Analysis of monitoring observations through 
+January 2013 shows that CTE continues to decline linearly over time 
+(`WFC3 ISR 2013-03 <https://www.stsci.edu/files/live/sites/www/files/home/
+hst/instrumentation/wfc3/documentation/instrument-science-reports-isrs/
+_documents/2013/WFC3-2013-03.pdf>`_).
+
+The CTE correction step uses its own dark reference files which have themselves 
+been corrected for CTE. The specific reference file used for any dataset may be 
+found in the image header keyword DRKCFILE. This step also uses a special 
+bias reference file as part of the CTE correction itself, referred to in the 
+header by the BIACFILE keyword. This BIACFILE is only used to facilitate the 
+CTE correction, the resulting corrected image then uses the normal BIASFILE to 
+correct the science frame after the CTE correction has been performed. After the 
+CTE correction has been performed and the data progresses through the rest of the 
+pipeline, the special CTE corrected dark, DRKCFILE in the header, will be used for 
+the dark current correction instead of the DARKFILE.
+
+There is a PCTETAB refrence file which contains extensions of calibration images 
+and tables of parameters used during the CTE correction stage. The header of this 
+file also contains parameters for the CTE correction algorithm. These parameters, 
+and important scalars which are used to correct the data are stored in the output 
+image headers. Users who wish to use other settings for the CTE correction algorithm 
+can adjust the pertinent keywords in their dataset header and the `wf3cte` component 
+of `calwf3` will give these values preference. See the 
+(`UVIS2.0 reference ISR <https://www.stsci.edu/files/live/sites/www/files/home/hst/
+instrumentation/wfc3/documentation/instrument-science-reports-isrs/
+_documents/2016/WFC3-2016-01.pdf>`_) 
+for more information regarding the parameter settings.  Please also see 
+the `ISR from 2021 <https://innerspace.stsci.edu/display/WFC3RSRC/
```

Comment:
this link to innerspace is not accessible to outside users - is there another?

Reply:
Addressed in 6101480.